src/settings-ui/Settings.UI.Library/ViewModels/ColorPickerViewModel.cs

```diff
@@ -56,7 +56,8 @@ public ColorPickerViewModel(ISettingsUtils settingsUtils, ISettingsRepository<Ge
                 { ColorRepresentationType.CIELAB, "CIE LAB - CIELab(76, 21, 80)" },
                 { ColorRepresentationType.CIEXYZ, "CIE XYZ - xyz(56, 50, 7)" },
                 { ColorRepresentationType.VEC4, "VEC4 - (1.0f, 0.7f, 0f, 1f)" },
-                { ColorRepresentationType.DecimalValue, "Decimal - 16755200" },
+                { ColorRepresentationType.DecimalValue, "Decimal BGR - 16755200" },
```

Comment:
if one is BGR and other is RGB, one of these two generated values would be wrong i'd imagine.

both can't be 16755200

Reply:
Looking at this I am a bit confused. Sure both formats can have the same decimal value but then it does not represent the same color.  I thought that it is only an example of the format, but should they be all the examples of value for the same color?  🤔 

If so, there are others that may not follow this pattern.
eg. cmyk(100%, 50%, 75%, 0%) hex: #ffaa00 vs rgb(100, 50, 75)
https://www.colorhexa.com/ffaa00

Reply:
in theory, (and i believe i have an open issue for this), the same RGB value should be used to calculate all the representations.  That is my hope.  After doing this, i think i'll go in and maybe do a PR to do just that :)

Reply:
Fixed to both examples represent #ffaa00 color